.github/workflows/build-db.yml

```
@@ -30,15 +30,19 @@ jobs:
         with:
           python-version: "3.10"
       - name: Install dependencies
+        id: install-dependencies
         run: |
           pip install --upgrade pip setuptools wheel build
+        background: true
```

actionlint (1.7.12)

[error] unexpected key "background" for step to run shell command. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "working-directory"

(syntax-check)